server/services/participants-helper/RegionsFilteredParticipantsFinder.ts

```diff
@@ -8,6 +8,9 @@ export type InterestFilter = { 'body.interested <>': string[] } | string[];
 export type IsIndigenousFilter = { 'body.isIndigenous =': boolean };
 export type idFilter = { 'id =': string };
 export type ProgramFilter = { 'body.program =': string };
+export type LivedLivingExperienceFilter = {
```

Comment:
I don't get the meaning of these filter types. The actual value passed as a param isn't an object of the types but just a string.